src/containers/Map/MapScreen.tsx

```diff
@@ -365,38 +359,36 @@ const formatEventBotsData = (
 			imgSrc: [Bot, Tank, Crane][idx % 3],
 		};
 
-		if (bot.status == 'InTransit') {
-			botPaths.push(bot.path);
-		}
-
 		botItems[bot._id] = items;
 	});
-	return { botArray: botMarkers, botHeaderInfo, botItems, botPaths };
+	return { botArray: botMarkers, botHeaderInfo, botItems };
 };
 
 const formatMapNodesData = (apiData: MapNode[]) => {
 	const mapNodeMarkers: { [key: string]: MarkerData } = {};
 	const mapNodeHeaderInfo: MapMenuProps['info'] = {};
 
-	apiData.forEach((node, idx) => {
-		// TODO: figure out what to name intermediate checkpoints
-		let name = node.name
-			? node.name
-			: 'Checkpoint ' +
-			  String.fromCharCode(65 + Math.floor(Math.random() * 26));
-		mapNodeMarkers[node._id] = {
-			_id: node._id,
-			name: name,
-			location: node.location,
-		};
-
-		mapNodeHeaderInfo[node._id] = {
-			topLeft: name,
-			topRight: node.distance.toFixed(0).toString() + 'm away',
-			bottomRight: node.eta.toFixed(1).toString() + ' minutes',
-			imgSrc: [LocationImgA, LocationImgB, LocationImgC][idx % 3],
-		};
-	});
+	apiData
+		.filter((node) => node.name)
+		.forEach((node, idx) => {
+			// TODO: figure out what to name intermediate checkpoints
+			let name = node.name
+				? node.name
+				: 'Checkpoint ' +
+				  String.fromCharCode(65 + Math.floor(Math.random() * 26));
+			mapNodeMarkers[node._id] = {
+				_id: node._id,
+				name: name,
+				location: node.location,
+			};
+
+			mapNodeHeaderInfo[node._id] = {
+				topLeft: name,
+				topRight: node.distance.toFixed(0).toString() + 'm away',
+				bottomRight: node.eta.toFixed(1).toString() + ' minutes',
+				imgSrc: [LocationImgA, LocationImgB, LocationImgC][idx % 3],
+			};
+		});
 	return { mapNodeArray: mapNodeMarkers, mapNodeHeaderInfo };
 };
 
```

Comment:
Do we need this function formatMapPathsData if we don't have paths anymore? Or we keeping it because we still need it for ordering a bot?

Reply:
keep it for ordering a bot